scripts/components/CollectionList.js

```diff
@@ -133,9 +135,11 @@ function ListActions(props) {
   return (
     <p className="actions">
       <button type="button"
-        className="btn-sync"
+        className="btn-sync btn btn-info"
         onClick={onSyncClick}>Synchronize</button>
-      <LinkButton label="Add" to={`/collections/${name}/add`} />
+      <span> </span>
```

Comment:
Yeah, need to remove them and style the spacing between elements using CSS here.